### DISCUSSION TITLE
pure javascript version

### DISCUSSION THREAD

**Discussion by scherler** [tailwindlabs/heroicons#617] [Ideas]
Hi there, awesome icons which I am using for my current development which is based on `lit` a web components library. I would love to be able to just import the lib as a web component. The idea is to expose something like `<heroicons type|name|...='plus' />` as a web component and not as vue or react since both are capable to use custom html elements. Would that sparkle some interest? If so I can provide a WC component as a wrapper.

**Comment by malobre** [comment]
For anyone interested, I made https://github.com/malobre/heroicons-wc for this exact purpose.
